Patches/Archotech PowerArmor/Apparel_JDS.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+    <Operation Class="PatchOperationFindMod">
+        <mods>
+            <li>Archotech PowerArmor</li>
+        </mods>
+        <match Class="PatchOperationSequence">
+            <operations>
+
+                <!-- ===== Add Bodypart Groups and Layers ===== -->
+                <li Class="PatchOperationAdd">
+                    <xpath>Defs/ThingDef[@Name="Trooper_HelmetBase"]/apparel/layers</xpath>
+                    <value>
+                        <li>OnHead</li>
+                        <li>StrappedHead</li>
+                    </value>
+                </li>
+
+                <li Class="PatchOperationAdd">
+                    <xpath>Defs/ThingDef[@Name="Trooper_ArmorBase"]/apparel/bodyPartGroups</xpath>
+                    <value>
+                        <li>Hands</li>
+                        <li>Feet</li>
+                    </value>
+                </li>
+
+                <!-- ===== Add Costlist ===== -->
+                <li Class="PatchOperationAdd">
+                    <xpath>Defs/ThingDef[@Name="Trooper_HelmetBase"]/costList</xpath>
+                    <value>
+                        <Uranium>20</Uranium>
+                        <Hyperweave>30</Hyperweave>
+                    </value>
+                </li>
+
+                <li Class="PatchOperationAdd">
+                    <xpath>Defs/ThingDef[@Name="Trooper_ArmorBase"]/costList</xpath>
+                    <value>
+                        <Uranium>45</Uranium>
+                        <Hyperweave>90</Hyperweave>
+                    </value>
+                </li>
+
+                <!-- ===== Stats Adjustment ===== -->
+
+                <!-- === Archotech Power Helmet === -->
+                <li Class="PatchOperationReplace">
+                    <xpath>Defs/ThingDef[@Name="Trooper_HelmetBase"]/statBases/WorkToMake</xpath>
+                    <value>
+                        <WorkToMake>26250</WorkToMake>
+                    </value>
+                </li>
+
+                <li Class="PatchOperationReplace">
+                    <xpath>Defs/ThingDef[@Name="Trooper_HelmetBase"]/statBases/MaxHitPoints</xpath>
+                    <value>
+                        <MaxHitPoints>360</MaxHitPoints>
+                        <Bulk>5.5</Bulk>
+                        <WornBulk>1.25</WornBulk>
+                        <ArmorRating_Heat>0.35</ArmorRating_Heat>

```

Comment:
Getting a duplicate of `ArmorRating_Heat`.

Reply:
good catch, I will be fixing it right away.